lib/config.py

```diff
@@ -59,9 +59,12 @@
         dict(help='Base Linux distribution version')
 }
 MOCK_ARGS = {
+    ('--mock-binary',):
+        dict(help='Mock binary path', default='/usr/bin/mock'),
```

Comment:
I prefer to remove all the default values from the command line parser in a separate patch